kubernetes-templates/weave-cloud/README.md

```diff
@@ -0,0 +1,23 @@
+# Weave Cloud Agents
+
+Weave Cloud is a add-on to Kubernetes which provides Continuous Delivery, along with hosted Prometheus Monitoring and a visual dashboard for exploring & debugging microservices.
+
+This package contains the agents which connect your cluster to Weave Cloud.
+
+_To learn more and sing up please visit [Weaveworks website](https://weave.works)._
```

Comment:
Just change the line 7 from
+_To learn more and sing up please visit [Weaveworks website]
to
+_To learn more and sign up please visit [Weaveworks website]